packages/grid_client/scripts/applications/funkwhale.ts

```diff
@@ -0,0 +1,121 @@
+import { FilterOptions, GatewayNameModel, MachinesModel } from "../../src";
+import { config, getClient } from "../client_loader";
+import { log, pingNodes } from "../utils";
+
+async function deploy(client, vms, subdomain, gatewayNode) {
+  const resultVM = await client.machines.deploy(vms);
+  log("================= Deploying VM =================");
+  log(resultVM);
+  log("================= Deploying VM =================");
+
+  const vmPlanetary = (await client.machines.getObj(vms.name))[0].planetary;
+  //Name Gateway Model
+  const gw: GatewayNameModel = {
+    name: subdomain,
+    node_id: gatewayNode.nodeId,
+    tls_passthrough: false,
+    backends: ["http://[" + vmPlanetary + "]:80"],
+  };
+
+  const resultGateway = await client.gateway.deploy_name(gw);
+  log("================= Deploying name gateway =================");
+  log(resultGateway);
+  log("================= Deploying name gateway =================");
+}
+
+async function getDeployment(client, vms, gw) {
+  const resultVM = await client.machines.getObj(vms.name);
+  const resultGateway = await client.gateway.getObj(gw);
+  log("================= Getting deployment information =================");
+  log(resultVM);
+  log(resultGateway);
+  log("https://" + resultGateway[0].domain);
+  log("================= Getting deployment information =================");
+}
+
+async function cancel(client, vms, gw) {
+  const resultVM = await client.machines.delete(vms);
+  const resultGateway = await client.gateway.delete_name(gw);
+  log("================= Canceling the deployment =================");
+  log(resultVM);
+  log(resultGateway);
+  log("================= Canceling the deployment =================");
+}
+
+async function main() {
+  const name = "newfunkwhale";
+  const grid3 = await getClient(`funkwhale/${name}`);
+  const subdomain = "fw" + grid3.twinId + name;
+  const instanceCapacity = { cru: 1, mru: 2, sru: 15 }; // Update the instance capacity values according to your requirements.
```

Comment:
Smallest package in Funkwhale uses 50 sru.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/702f3d37-8e1d-4f21-a8c4-c7aa147670dc)